### ISSUE TITLE
README bgpkit broker example erroneous

### ISSUE THREAD

**Issue by egecemkirci** [bgpkit/pybgpkit#4]
Two points regarding the README example confuse me:

First, the `assert len(items) == 58` does not hold (106 items return) when I run the example as is. Second, when I delve into the details of returned items, I see MRT files starting from before `ts_start` and ending after `ts_end`; which makes me think if I misinterpret these arguments. As in the given example, these two arguments refer to the same time, I would actually expect 0 MRT files to be returned (as none would start and end simultaneously). 

For the reference:

```
import bgpkit
broker = bgpkit.Broker()
items = broker.query(ts_start="1634693400", ts_end="2021-10-20T01:30:00")
for item in items:
    print(item)
print(len(items))
assert len(items) == 58
```

**Comment by digizeph** [comment]
Thanks for the comment! There are places in the documentation that needs some more love.

For the issues you encountered, the explanation is this: the `ts_start` and `ts_end` describes the *inclusive* time range where you would want to search the BGP messages for. The BGPKIT Broker returns any files that *might* contain BGP messages that originated within that time range. If the `ts_start` falls on the border of the data intervals (i.e., 15min for RV and 5min for RIS), it will include the files for the previous time bin because it may include messages at `ts_start`. 

In summary, bgpkit broker searches for files that might contain relevant BGP messages.

This was changed after the initial release, and we still had the older version of the `assert` statement that you saw. It's half of the value for exactly this reason (the additional files were from the previous time bin).

You might do `ts_start + 1` and `ts_end -1` to get only the files within the exclusive time range, and not the files after or before if `ts_start` and `ts_end` were on the 15min or 5min marks.

**Comment by egecemkirci** [comment]
Thanks a lot for the explanation!
